src/samplers/orthogonal.cpp

```diff
@@ -0,0 +1,225 @@
+#include <mitsuba/core/profiler.h>
+#include <mitsuba/core/properties.h>
+#include <mitsuba/core/spectrum.h>
+#include <mitsuba/render/sampler.h>
+
+NAMESPACE_BEGIN(mitsuba)
+
+/**!
+
+.. _sampler-orthogonal:
+
+Orthogonal Array sampler (:monosp:`orthogonal`)
+-----------------------------------------------
+
+.. pluginparameters::
+
+ * - sample_count
+   - |int|
+   - Number of samples per pixel. This value has to be the square of a prime number. (Default: 4)
+ * - strength
+   - |int|
+   - Orthogonal array's strength (Default: 2)
+ * - seed
+   - |int|
+   - Seed offset (Default: 0)
+ * - jitter
+   - |bool|
+   - Adds additional random jitter withing the substratum (Default: True)
+
+This plugin implements the Orthogonal Array sampler generator introduced by
+:cite:`jarosz19orthogonal`. It generalizes correlated multi-jittered sampling to higher dimensions
```

Comment:
Technical writing tip: never use a citation as a word in a sentence. It is metadata that is not pronounced when reading the text. Better: `Jaroz et al. [citation link]`